cmd/models/onebuild-execution-plan.go

```diff
@@ -23,6 +25,13 @@ type CommandContext struct {
 	CommandSession *sh.Session
 }
 
+const (
+	commandBannerLength = 72
```

Comment:
length 72 will be constant across the project – as a guideline.

Can we extract this as constant in `utilis.go`?